app/uk/gov/hmrc/cdsreimbursementclaimstubs/models/acc14/Acc14Response.scala

```diff
@@ -181,6 +190,158 @@ object Acc14Response {
+         |                        "telephone": "0270 112 3476",
+         |                        "emailAddress": "fred@bloggs.com"
+         |                    }
+         |                },
+         |                {
+         |                    "accountType": "002",
+         |                    "accountNumber": "8901113",
+         |                    "eori": "8432563",
+         |                    "legalName": "Fred Bloggs and Co Ltd",
+         |                    "contactDetails":
+         |                    {
+         |                        "contactName": "Angela Smith",
+         |                        "addressLine1": "J P Jones Insolvency Ltd",
+         |                        "addressLine2": "14 Briar Lane",
+         |                        "addressLine3": "London",
+         |                        "countryCode": "GB",
+         |                        "telephone": "0270 112 3476",
+         |                        "emailAddress": "fred@bloggs.com"
+         |                    }
+         |                }
+         |            ],
+         |            "securityDetails":
+         |            [
+         |                {
+         |                    "securityDepositId": "ABC0123456",
+         |                    "totalAmount": "14585.52",
+         |                    "amountPaid": "14585.52",
+         |                    "paymentMethod": "001",
+         |                    "paymentReference": "SGL SECURITY 001",
+         |                    "taxDetails":
+         |                    [
+         |                        {
+         |                            "taxType": "A00",
+         |                            "amount": "6000.00"
+         |                        },
+         |                        {
+         |                            "taxType": "B00",
+         |                            "amount": "8085.52"
+         |                        }
+         |                    ]
+         |                },
+         |                {
+         |                    "securityDepositId": "DEF6543210",
+         |                    "totalAmount": "500.00",
+         |                    "amountPaid": "300.00",
+         |                    "paymentMethod": "004",
+         |                    "paymentReference": "SGL SECURITY 002",
+         |                    "taxDetails":
+         |                    [
+         |                        {
+         |                            "taxType": "A00",
+         |                            "amount": "100.00"
+         |                        },
+         |                        {
+         |                            "taxType": "B00",
+         |                            "amount": "200.00"
+         |                        }
+         |                    ]
+         |                }
+         |            ]
+         |        }
+         |    }
+         |}
+         |""".stripMargin)
```

Comment:
I wish there was a more cleaner way to do this, like Acc14ReponseBuilder or something